pkg/sink/codec/internal/message_key.go

```diff
@@ -28,6 +28,8 @@ type MessageKey struct {
 	RowID     int64             `json:"rid,omitempty"`
 	Partition *int64            `json:"ptn,omitempty"`
 	Type      model.MessageType `json:"t"`
+	// Only Handle Key Columns encoded in the message's value part.
+	OnlyHandleKey bool `json:"ohk,omitempty"`
```

Comment:
`ohk` is a not good name,  why not use full name?

Reply:
follow the open-protocol naming convention.